test/Microsoft.AspNet.Mvc.Razor.Host.Test/ModelChunkVisitorTest.cs

```diff
@@ -105,7 +106,8 @@ private static CodeBuilderContext CreateContext()
                                          "MyClass",
                                          "MyNamespace",
                                          string.Empty,
-                                         shouldGenerateLinePragmas: true));
+                                         shouldGenerateLinePragmas: true),
+                new ParserErrorSink());
```

Comment:
:-1: this is the second parameter to to the `CodeGeneratorContext` constructor, not an additional parameter to the `MvcRazorHost` constructor